### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by johnpeart**
I found two of what I think are typos in the README file.
This README file fixes the typos.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,8 +34,8 @@ python3 -m venv /path/to/new/virtual/environment
 source /path-to-new-virtual-environment/bin-activate
 ```
 
-Now install the requirements for this project: `pip install -r requirements.txt`
+Now install the requirements for this project: `pip3 install -r requirements.txt`
```
Comment (jonodrew) [inline README.md:37]: This isn't right - if the virtual environment is set up with python3, then the version of pip should be correct

File: README.md [hunk 2]
```diff
@@ -34,8 +34,8 @@ python3 -m venv /path/to/new/virtual/environment
...
 
 This software is best run in Docker, because it needs Redis and a Celery worker. You'll need to install `Docker` and
 `docker-compose`. [Docker instructions](https://docs.docker.com/engine/install/) and
-[docker-compose instructions](https://docs.docker.com/compose/install/). Once you've got those two installed, you'll need to run `docker-compose -up -d` to get things running, and `docker-compose down` to stop them again. You should see the app running at http://127.0.0.1:5001
+[docker-compose instructions](https://docs.docker.com/compose/install/). Once you've got those two installed, you'll need to run `docker-compose up -d` to get things running, and `docker-compose down` to stop them again. You should see the app running at http://127.0.0.1:5001
```
Comment (jonodrew) [inline README.md:41]: Well spotted, thank you!

**Review by jonodrew — CHANGES_REQUESTED** [review]

**Comment by johnpeart** [comment]
I've rolled back the `pip3` change; it failed without it on my local set up but I suspect I had done something wrong!

**Comment by jonodrew** [comment]
Very interesting! What error message do you get?

**Comment by johnpeart** [comment]
I didn't note it down at the time; so not entirely sure how to recreate it now!

I'm not even sure at this point if the steps before the Docker install are necessary, as the install there seemed to run without errors. As I say – very likely I was doing something wrong so stick with your gut!

**Comment by jonodrew** [comment]
🤔 alright, let's get this merged. I hate to leave a problem unsolved!

**Review by jonodrew — APPROVED** [review]
Looks good 👍🏻  please could you merge it in?

**Pull request merged**